examples/pet-store-sse/src/web-server.ts

```
@@ -1,286 +1,304 @@
-    } catch (error) {
-      console.error(`Error connecting transport for session ${sessionId}:`, error);
+
+      // Make the transport available to the MCP server
+      try {
+        transport.onmessage = async (message: JSONRPCMessage) => {
+          try {
+            // The server will automatically send a response via the transport
+            // if the message has an ID (i.e., it's a request, not a notification)
+          } catch (error) {
+            console.error('Error handling MCP message:', error);
+          }
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)